investment-app/TRINV.Api/TRINV.Infrastructure/TRINV.Infrastructure.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.25" />
```

Comment:
@glachezar I think we will need and Microsoft.EntityFrameworkCore.Design

Reply:
We will need it. You are absolutely right.